src/client/Inode.h

```diff
@@ -211,7 +211,7 @@ struct Inode : RefCountedObject {
   int cache_gen = 0;
   int snap_caps = 0;
   int snap_cap_refs = 0;
-  utime_t hold_caps_until;
+  auto hold_caps_until;
```

Comment:
We don't use `auto` in a header file for the type. This should reference the `ceph::coarse_mono_time` type. See how we do that in `src/mds` for examples.

Reply:
Made changes

Reply:
This also needs fix where `hold_caps_until` is being used, as it's type changed.

Reply:
Done